commands/settings.js

```
@@ -1,91 +1,154 @@
-const settings = require("../utils/settings.js");
-const embed = require("../utils/embed.js");
+const { Permissions } = require('discord.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const settings = require('../utils/settings.js');
+const embed = require('../utils/embed.js');
 
-const idResolver = (collection, id) =>  {
+const idResolver = (collection, id) => {
 	const item = collection.get(id);
-	if (item) return (item.name);
-	return "ERROR";
+	if (item) return item.name;
+	return 'ERROR';
```

should this throw an errior instead of returning a string :?